types/node/child_process.d.ts

```diff
@@ -230,7 +230,7 @@ declare module "child_process" {
          * The `subprocess.spawnargs` property represents the full list of command-line
          * arguments the child process was launched with.
          */
-        readonly spawnargs: string[];
+        readonly spawnargs: readonly string[];
```

Comment:
Sorry, I thought this was an options parameter at a glance, but it's obviously a class instance property. This shouldn't be changed; not only does it not match runtime behaviour, but it breaks usage like
```ts
const args: string[] = cp.spawnargs;
```
which becomes a compiler error.